tools/create-book.cmd

```diff
@@ -1,12 +1,50 @@
 @echo off
```

Comment:
There were two copies of this file.  I used the one that was most recently committed to.